README.md

```diff
@@ -263,6 +263,22 @@ Once these are in place, you will be able to run the following:
 $ cmr test cicd
 ```
 
+Alternatively to run only certain types of tests, you may run the following:
+
+Unit Tests:
+```
+lein modules utest
+```
+
+Integration Tests against a running CMR system:
+```
+# If running CMR with the in-memory database (default)
+lein modules itest --skip-meta :oracle
```

Comment:
Do you have these commands swapped?

Reply:
skip-meta will ignore anything with the tags provided. The command will skip tests requiring a live connection.

Reply:
where is the full list of tags?

Reply:
Those are the only two tags available that are specific to CMR. Kaocha has :pending and :skip available but those aren't important to a user running the tests.

Reply:
I think the tests you can run should be listed, new developers will not know what is possible.